README.md

```diff
@@ -137,8 +137,7 @@ github pages.
 ## 🔭 Future Features <a name="future-features"></a>
 
 
-
-- **[API]**-> Allow submit and refresh from API.
+-style the web page.
 
 
```

Comment:
### CHANGES TO TEST AND INSTALL

**TESTS**
- [ ] Please, kindly remove the test section since the documentation is not relevant to this project. Also, you won't be writing any test for this project so, let's kindly remove it and also from the navigation in the top part of the readme to make the readme more professional.

**INSTALL**

- [ ] Hey, instead of writing what users can install manually, kindly tell the users to run the `npm install` command as this would install all dependencies used in the project in one line of command. please, update the section.